src/main/java/in/zapr/druid/druidry/aggregator/DoubleMaxAggregator.java

```diff
@@ -19,17 +19,28 @@
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NonNull;
+import lombok.Setter;
 
 @Getter
 @EqualsAndHashCode(callSuper = true)
 public class DoubleMaxAggregator extends DruidAggregator {
 
     private static final String DOUBLE_MAX_TYPE_AGGREGATOR = "doubleMax";
+
     private String fieldName;
 
+    @Setter
+    private String expression;
+
+    public DoubleMaxAggregator(@NonNull String name) {
```

Comment:
using constructor for a field and then setting for another field is not a good idea. We can expose builder and mark old constructor as deprecated. This way it's not a breaking change and also new `expression` functionality is integrated.